spec/java/util/OptionalLong.lsl

```diff
@@ -0,0 +1,282 @@
+libsl "1.1.0";
+
+library "std:collections"
+    version "11"
+    language "Java"
+    url "-";
+
+// imports
+
+import "java-common.lsl";
+import "java/lang/_interfaces.lsl";
+import "java/util/function/_interfaces.lsl";
+import "java/util/stream/_interfaces.lsl";
+
+
+// local semantic types
+
+
+
+// automata
+
+@WrapperMeta(
+    src="java.util.OptionalLong",
+    dst="ru.spbpu.libsl.overrides.collections.OptionalLong",
+)
+@public automaton OptionalLong: long
+(
+    var value: long = 0;
+    var present: boolean = false;
+)
+{
+    // states and shifts
+
+    initstate Allocated;
+    state Initialized;
+
+    shift Allocated -> Initialized by [
```

Comment:
I didn't find some of this methods (ofNullable ,filter, map,...) in class "OptionalLong" and in this automaton created methods too.

Reply:
Fixed